docs/data-sources/remote-mcp-servers.md

```
@@ -0,0 +1,176 @@
+MCP currently supports three transport mechanisms: stdio, Server-Sent Events (SSE), and Streamable HTTP.
+At this time, HolmesGPT is compatible only with MCP servers that use SSE.
+However, many existing MCP servers—such as Dynatrace MCP—rely exclusively on the stdio transport.
+To overcome this incompatibility, tools like Supergateway can act as a bridge by converting stdio-based MCPs into SSE-compatible endpoints.
+
+For this demo we will use:
+- [Dynatrace MCP](https://github.com/dynatrace-oss/dynatrace-mcp)
+- [Supergateway](https://github.com/supercorp-ai/supergateway) - runs MCP stdio-based servers over SSE
+
+Check out supergatway docs to find out other useful flags.
+
+**See it in action**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)